monitoring/monitoring-api/src/main/java/com/rbmhtechnology/vind/monitoring/MonitoringSearchServer.java

```diff
@@ -214,7 +214,7 @@ public void commit(boolean optimize) {
         return execute(search,c,this.session);
     }
 
-    public <T> BeanSearchResult<T> execute(FulltextSearch search, Class<T> c, Session session) {
+    protected <T> BeanSearchResult<T> execute(FulltextSearch search, Class<T> c, Session session) {
```

Comment:
I think this should stay `public` - otherwise it's a change in the public api...
```suggestion
    public <T> BeanSearchResult<T> execute(FulltextSearch search, Class<T> c, Session session) {
```